oracle/oracle-explanation/canonical-offerings.rst

```diff
@@ -0,0 +1,62 @@
+Canonical's offerings on Oracle Cloud
+=====================================
+
+Ubuntu images
+-------------
+
+Canonical produces fully optimised Oracle Cloud images that provide a stable base image for running workloads. The two main types of images are:
+
+* **Server images:** General-purpose customised images that include the necessary drivers and kernel modifications to run smoothly in the Oracle Cloud environment. These images launch with the oracle-cloud-agent pre-installed, featuring plugins that collect performance metrics, install OS updates, and perform other instance management tasks.
+
+* **Minimal images:** These are minimal images designed for automated deployment at scale and have a reduced default package set. They are much smaller, boot faster, and require fewer security updates over time due to the fewer installed packages.
```

Comment:
To me (maybe that's not how it is describe elsewhere) it sounds strange to distinguish between Server and Minimal image in this way. Minimal images **are** server image but just stripped down to the bare minimal.
Also, there is a repetition here. We could probably just write

```suggestion
* **Minimal images:** These are images designed for automated deployment at scale and have a reduced default package set. They are much smaller, boot faster, and require fewer security updates over time due to the fewer installed packages.
```

Reply:
Just for clarification, I would prefer something like "Full Server Image" vs "Minimized Server Images"

Reply:
I agree with @gjolly, but I would like to point out that, in Oracle Cloud console, the names of the images are "Canonical Ubuntu <release>", "Canonical Ubuntu <release> Minimal" and "Canonical Ubuntu <release> Minimal aarch64", so if we want to be consistent with what the user would see in the cloud platform, we should stick to "minimal" instead of "minimized"

Reply:
Elsewhere we use 'Server images' and 'Minimal server images'. Will that work? 
(Also, I have removed the repeated 'minimal')

Reply:
It's fine for me. As long as we make it clear that it's also a server image I am fine.